### DISCUSSION TITLE
How to access app without paywall?

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#1590] [Help needed]
I noticed the F-droid version was not getting updates, so I though I will do a manual GitHub install. I made a backup and after installing the GitHub .apk (any version for that matter) I noticed that there is a paywall. So now I'm unable to access my subscriptions filters or any other function. How do I access the app, or at least my data without having to dig through the backup file?

**Comment by SirenGlitch** [comment]
https://colab.research.google.com/drive/13AE8RvjnCfuBJGaACEqxeBIMo33_l-Sc?usp=sharing#scrollTo=BNM9Hdc_iHNX
